buildSrc/src/main/groovy/io.deephaven.hadoop-common-dependencies.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'java'
+}
+
+dependencies {
+    implementation(libs.hadoop.common) {
```

Comment:
I didn't realize this had moved to Classpaths - I don't love that it's a plugin, but better than repeating it in two places.